domains/libraries/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation/B2C/AccessDescriptor.cs

```diff
@@ -14,7 +14,8 @@ public AccessDescriptor(IdentityDescriptor identity)
     public bool IsAuthorizedToOrganization(Guid organizationId)
     {
         var isInternalClient = _identity.SubjectType == SubjectType.Internal;
+        var isOwnOrganization = _identity.OrganizationId == organizationId;
         var isAuthorizedToOrganization = _identity.AuthorizedOrganizationIds.Contains(organizationId);
-        return isInternalClient || isAuthorizedToOrganization;
+        return isInternalClient || isOwnOrganization || isAuthorizedToOrganization;
```

Comment:
Du har tabt mig her. Hvad betyder det at man er OwnOrg?

Reply:
The _isOwnOrganization_ variable is true if _organizationId_ is the _id_ of the _Organization_ the user is affiliated with.

So you are authorized to act on behalf of the organization identified by organizationId **if**:

You are an internal client (**us**)
**or**
You are affiliated with the organization (You chose that organization when logging into MitID)
**or**
You have been granted consent to the organization (A user from organization granted your organization consent)